### ISSUE TITLE
Finetune 2.8B model on chemrxiv

### ISSUE THREAD

**Issue by jackapbutler** [OpenBioML/chemnlp#171]
* blocked by outcomes of #170

This will include 
1) finding the optimal pipeline configuration for this model size
2) training this model on the cluster using this configuration for full model finetuning
3) saving these checkpoints so that the evaluation pipeline can process them

We want to get this model to the stage where others can iterate on training aspects such as learning rate scheduler, optimiser, number of epochs, etc with training efficiency staying high.

**Comment by jackapbutler** [comment]
This has been deprioritised in favour of understanding tools and speed estimates for scaling to large 10's B models.

**Comment by jackapbutler** [comment]
A first training run was completed but showed the model failing to converge. The results and hyperparameters are available [here](https://www.notion.so/stable-society/Pythia-2-8B-FT-v0-crashed-bad-bfdae057263c46cea7d99541f0e04cb7). I'm going to debug this further now.
